src/main.ts

```diff
@@ -0,0 +1,45 @@
+import { app } from 'electron';
+import { emptyDirSync } from 'fs-extra';
+import path from 'path';
+
+const relaunchApp = () => {
+  app.relaunch();
+  app.exit();
+};
+
+/**
+ * Removes the contents of the user data directory.
+ * This directory is then recreated after re-launching the application.
+ */
+function clearUserDataDirectory(): void {
```

Comment:
```suggestion
export function clearUserDataDirectory(): void {
```
I like putting the export keyword right at the declaration site, because it works like a visibility modifier (`public`, `private` etc.)
What do you think?